Core/MIPS/RiscV/RiscVCompALU.cpp

```diff
@@ -232,7 +232,41 @@ void RiscVJitBackend::CompIR_Bits(IRInst inst) {
 
 	switch (inst.op) {
 	case IROp::ReverseBits:
-		CompIR_Generic(inst);
+		if (cpu_info.RiscV_Zbb) {
+			gpr.MapDirtyIn(inst.dest, inst.src1);
+			// Start by reversing bytes (note: this puts in upper 32 of XLEN.)
+			REV8(gpr.R(inst.dest), gpr.R(inst.src1));
+
+			// Swap nibbles.
```

Comment:
Huh, I was really surprised to see that RISC-V is lacking a bit reverse instruction. After some googling, it seems it has existed in proposals in a couple of forms but got cut, heh. Oh well.